portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Projects</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
```

Comment:
consider creating a new JS file for the projects page specifically

Reply:
I believe the projects page is the only page that uses the JS file. Would it be good practice to simply remove `<script src="script.js>` from all other pages?